### DISCUSSION TITLE
Automatic generation of subtitles with AI

### DISCUSSION THREAD

**Discussion by caJosue0060** [SubtitleEdit/subtitleedit#6717] [Ideas]
Recently the people of OpenAI released the Whisper model, an excellent transcription system in a large number of languages.
https://openai.com/blog/introducing-chatgpt-and-whisper-apis

With this API it would be interesting to add a button that automatically generates the subtitles (.str) of the video we are dealing with. I have no idea how it would be done, but it does not seem to be such a complicated job and it would make a lot of work easier.

**Comment by darnn** [comment]
This has already been implemented. Download the latest beta:
https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.11/SubtitleEditBeta.zip
Go to the Video menu, and there to Audio to text (Whisper). Then in this window click the text in the upper right corner and select Const-Me.
